src/sentry/toolbar/views/iframe_view.py

```diff
@@ -1,19 +1,23 @@
+import logging
 from typing import Any
 
 from django.http import HttpRequest, HttpResponse
 
 from sentry.models.organization import Organization
 from sentry.models.project import Project
-from sentry.toolbar.utils.url import check_origin
-from sentry.web.frontend.base import OrganizationView, region_silo_view
+from sentry.toolbar.utils.check_origin import check_origin
+from sentry.toolbar.views.base import ToolbarView
+from sentry.web.frontend.base import region_silo_view
 
 REFERRER_HEADER = "HTTP_REFERER"  # 1 R is the spelling used here: https://docs.djangoproject.com/en/5.1/ref/request-response/
 SUCCESS_TEMPLATE = "sentry/toolbar/iframe.html"
 INVALID_TEMPLATE = "sentry/toolbar/iframe-invalid.html"
 
+logger = logging.getLogger(__name__)
+
 
 @region_silo_view
-class IframeView(OrganizationView):
+class IframeView(ToolbarView):
     def respond(self, template: str, context: dict[str, Any] | None = None, status: int = 200):
```

Comment:
This class is overriding the respond method in the base class.  Should it be moved out of the base class and into the login view?

Reply:
We only need the `X-Frame-Options` for the iframe response.

Reply:
Oh I see I missed the call to super.  👍